src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -0,0 +1,263 @@
+    rowGap: theme.spacing(3),
+    [theme.breakpoints.down('xs')]: {
+      gridTemplateColumns: '1fr',
+      gridAutoRows: '1fr',
+    },
+  },
+  sectionTitle: {
+    color: theme.colors.primary,
+    size: 20,
+    lineHeight: 1,
+    fontWeight: 400,
+    margin: theme.spacing(6, 0, 0),
+  },
+  sectionSubtitle: {
+    color: theme.colors.primary,
+    textAlign: 'center',
+    size: 10,
+    lineHeight: 1.55,
+    fontWeight: 300,
+  },
+  longform: {
+    width: '100%',
+    margin: theme.spacing(2, 0),
+    maxWidth: 800,
+    '& > *': {
+      margin: theme.spacing(0, 0, 3),
+    },
+    '& label': {
+      alignSelf: 'flex-start',
+    },
+  },
+  saveButton: {
+    margin: theme.spacing(3, 0, 5),
+  },
+  errors: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-end',
+    margin: 0,
+    minHeight: 45,
+    color: theme.colors.red,
+  },
+}));
+
+export const SummonCirclePage = () => {
+  const classes = useStyles();
+  const myAddress = useMyAddress();
+  const myAdminCircles = useMyAdminCircles();
+
+  const protocols = useMemo(
+    () =>
+      uniqBy(
+        myAdminCircles.map(({ protocol }) => protocol),
+        'id'
+      ),
+    [myAdminCircles]
+  );
+
+  const { createCircle } = useApi();
+
+  if (myAddress === undefined) {
+    return <></>;
+  }
+
+  return (
+    <div className={classes.root}>
+      <h2 className={classes.title}>Create a Circle</h2>
+      <CreateCircleForm.FormController
+        source={undefined}
+        hideFieldErrors
+        submit={({
+          h_captcha_token,
+          research_what_needs,
+          research_how_use,
+          research_org_structure,
+          research_org_link,
+          research_org_ens,
+          research_contact,
+          ...params
+        }) =>
+          createCircle(
+            { address: myAddress, ...params },
+            h_captcha_token,
+            JSON.stringify({
+              address: myAddress,
+              research_what_needs,
+              research_how_use,
+              research_org_structure,
+              research_org_link,
+              research_org_ens,
+              research_contact,
+              ...params,
+            })
+          )
+        }
+      >
+        {({ fields, errors, changedOutput, handleSubmit }) => (
+          <div className={classes.bodyInner}>
+            <div className={classes.subtitle}>
+              Coordinape circles allow you to collectively reward circle members
+              through equitable and transparent payments. To start a circle, we
+              need just a bit of information.
+            </div>
+            <div className={classes.twoColumnGrid}>
+              <FormTextField {...fields.user_name} label="Username" fullWidth />
+              <ApeTextField
+                label="ETH Address"
+                value={myAddress}
+                disabled
+                fullWidth
+              />
+              {myAdminCircles ? (
```

Comment:
you need to check emptiness, not existence, here. currently "Select from list to add to existing org" shows up even when a wallet has no circles

Reply:
Good catch

Reply:
My aversion to implicit truthy conversion wouldn't have saved me here, using `!!`